src/App.tsx

```diff
@@ -26,70 +27,127 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((prevGood, nextGood) => {
+    if (sortType === SortType.ALPHABET) {
+      return prevGood.localeCompare(nextGood);
+    }
+
+    if (sortType === SortType.LENGTH) {
+      return prevGood.length - nextGood.length;
+    }
+
+    return 0;
+  });
```

Comment:
Use switch case
Check the sort type before sort
Don't sort array if there is no sortType

```suggestion
  switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```

Reply:
done